packages/cmf/src/bootstrap.md

```diff
@@ -45,18 +49,19 @@ You apps may be splitted into different modules, like a core set of components a
 For that CMF expose an API to let you merge your settings:
 
 ```javascript
+// /src/app/index.js
 import cmf from '@talend/react-cmf';
-
 import containersModule from '@talend/react-containers';
 import components from './components';
-import saga from './saga';
 import sagas from './sagas';
 
-cmf.boostrap({
+cmf.bootstrap({
 	components,
-	saga,
-	sagas,
-	modules: [containersModule],
+	saga: sagas.appSaga,
```

Comment:
?? I don t understand this. 
The main saga should never leave in sagas. You will not do DI with it

Reply:
Can you update the 2 examples with what you think is appropriate about folder structure/export style (default/named, things grouped or not) about the sagas ?